annotator/static/views/player.js

```diff
@@ -46,6 +46,12 @@ class PlayerView {
         // Video URL
         this.videoSrc = videoSrc;
 
+        // Video start time
+        this.videoStart = parseFloat(videoStart);
```

Comment:
I think the solution is to parse it in the backend, not in the frontend. Later the backend won't be parsing it from a query string so the frontend can assume it's a float